package.json

```diff
@@ -41,6 +46,7 @@
   },
   "dependencies": {
     "distributions-normal-random": "0.0.1",
+    "fs": "0.0.1-security",
```

Comment:
please no. fs is bundled in node. Don't install it from npm

Reply:
thanks I did not kwon that